pom.xml

```diff
@@ -218,7 +218,7 @@
     <dependency>
       <groupId>com.jayway.jsonpath</groupId>
       <artifactId>json-path</artifactId>
-      <version>2.5.0</version>
+      <version>2.6.0</version>
```

Comment:
@raul-arabaolaza BTW, what made you go for 2.6.0 when there's a 2.7.0 since January 2022 AFAICT?
https://search.maven.org/artifact/com.jayway.jsonpath/json-path/2.7.0/jar

Reply:
though then I'm confused as to why there wouldn't be a dependabot PR already for it... :thinking:

Reply:
> @raul-arabaolaza BTW, what made you go for 2.6.0 when there's a 2.7.0 since January 2022 AFAICT? https://search.maven.org/artifact/com.jayway.jsonpath/json-path/2.7.0/jar

Given I did not want any new feature I went with the smallest possible update to reduce the possibility of binary incompatibilities with the existing code. A general update strategy for dependencies should be done in separate PRs